src/api/payment-requests/payment-requests.md

```diff
@@ -98,7 +98,7 @@ version (documented on this page) and the "legacy" version (documented at
 
 ★  For payment options which specify an address, there's a requirement to make a transaction on an
 external ledger. Once you have made that payment, you can use the transaction id for
-[Paying a Payment Request][].
+[Paying a Legacy Payment Request][].
```

Comment:
I think this needs rewording. Legacy belongs in here, but maybe not in the link.